rest/middleware.go

```diff
@@ -130,7 +130,7 @@ func createEvgAuthRequest(ctx context.Context, r *http.Request, evgConf *model.E
 	urlString := fmt.Sprintf("%s/rest/v2/auth?resource=%s&resource_type=project&permission=project_logs&required_level=10", evgConf.URL, resourceID)
 	req, err := http.NewRequest(http.MethodGet, urlString, nil)
 	if err != nil {
-		return nil, gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "error creating http request"))
+		return nil, gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "creating http request"))
```

Comment:
`HTTP`

Reply:
Fixed. Also fixed capitalization in all other strings/comments I could find.